tsconfig.json

```diff
@@ -14,6 +14,9 @@
         "resolveJsonModule": true,
         "isolatedModules": true,
         "noEmit": true,
+        "noEmitOnError": true,
```

Comment:
These changes may not be necessary, and we may close this PR, depending on the results of #214